analyzer/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG RELEASE=bullseye
```

Comment:
I'd suggest using bullseye slim, and if possible alpine ? I know that currently with liquidsoap /gstreamer/... using alpine is somewhat difficult, hence #1470 and #1221. But I would suggest we keep this in mind.

Reply:
Liquidsoap does actually have a alpine package https://github.com/savonet/liquidsoap/issues/1646

Reply:
I really dislike the alpine images, simply because they are very different and often cause issues (like PATH search failures). The Debian images are well known and easy to modify. Also all our tooling around package names is centered around Debian/Ubuntu. I want to re-use that instead of adding an additional package name for all the alpine things that then change and get dropped very rapidly from their archive.

Reply:
Yes, I get what you mean. Maybe we should consider this only once we have fully working images and less dependencies. I think having lightweight containers is still a good thing. But should not generate headache.